src/tokenizer.rs

```diff
@@ -144,14 +150,15 @@ impl<'o, A: AsRef<[u8]>> Tokenizer<'_, A> {
 ///
 pub struct TokenizerBuilder<'sw, A> {
     stop_words: Option<&'sw Set<A>>,
+    create_char_map: bool,
 }
 
 impl<'sw, A> TokenizerBuilder<'sw, A> {
     /// Create a `TokenizerBuilder` with default settings,
     ///
     /// if you don't plan to set stop_words, prefer use [`TokenizerBuilder::default`]
     pub fn new() -> TokenizerBuilder<'sw, A> {
-        Self { stop_words: None }
+        Self { stop_words: None, create_char_map: true }
```

Comment:
If you directly wrap `create_char_map` it in a `NormalizerOption`, you can do:
```suggestion
        Self { stop_words: None, normalizer_option: NormalizerOption::default() }
```